helix-view/src/handlers/dap.rs

```diff
@@ -393,10 +438,66 @@ impl Editor {
                             shell_process_id: None,
                         }))
                     }
+                    Ok(Request::StartDebugging(arguments)) => {
+                        let debugger = match self.dap_servers.get_client_mut(id) {
+                            Some(debugger) => debugger,
+                            None => {
+                                self.set_error("No active debugger found.");
+                                return true;
+                            }
+                        };
+                        // Currently we only support starting a child debugger if the parent is using the TCP transport
+                        let socket = match debugger.socket {
+                            Some(socket) => socket,
+                            None => {
+                                self.set_error("Child debugger can only be started if the parent debugger is using TCP transport.");
+                                return true;
+                            }
+                        };
+
+                        let config = debugger.config.clone();
+
+                        let result = self
+                            .dap_servers
+                            .start_client(Some(socket), &config.expect("No config found"));
```

Comment:
Instead of an `expect` let's `log::error!` when the config isn't set and `return true`